api/proto/teleport/resourceusage/v1/resourceusage_service.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.resourceusage.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/resourceusage/v1;resourceusagev1";
+
+// AccessRequestUsage defines the usage limits for access requests.
+// Currently this is limited on the basis of access requests used per calendar month.
+message AccessRequestUsage {
+  // MonthlyLimit is the amount of requests that are allowed per month
+  int32 monthly_limit = 1;
+  // MonthlyUsed is the amount of requests that have been used this month
+  int32 monthly_used = 2;
+}
+
+// GetAccessRequestUsageRequest is the request for GetResourceUsage.
+message GetAccessRequestUsageRequest {}
+
+// ResourceUsageService is a service to fetch information about usage of limited resource on usage-billed plans.
+service ResourceUsageService {
+  // GetAccessRequestUsage returns the usage of access requests
+  rpc GetAccessRequestUsage(GetAccessRequestUsageRequest) returns (AccessRequestUsage);
```

Comment:
Sorry, I missed this in the other round.

Could we make this RPC into a generic "GetResourceUsage" RPC, so we can aggregate the various usage metrics under it?

One of the reasons we are doing this it to not have to do a single RPC for every usage metric we want to display in the billing summary page.